Makefile

```diff
@@ -1,13 +1,34 @@
-.PHONY: com.docker.slirp.exe install uninstall
+.PHONY: com.docker.slirp.exe com.docker.slirp install uninstall
+
+TARGETS :=
+ifeq ($(OS),Windows_NT)
+	TARGETS += com.docker.slirp.exe
+else
+	TARGETS += com.docker.slirp
+endif
+
+all: $(TARGETS)
 
 EXEDIR=C:\projects\vpnkit
 
 com.docker.slirp:
-	cd src/com.docker.slirp && \
-	oasis setup && \
-	./configure && \
-	make && \
-	cp _build/src/main.native com.docker.slirp
+	$(MAKE) -C src/com.docker.slirp
+
+depends:
+	opam repo list | grep com-docker-slirp || opam repo add com-docker-slirp .
```

Comment:
True, but I don't generally run the stuff in arbitrary `.yml` files in my development environment without having to copy them out to a terminal or otherwise look at them first, giving me an opportunity to say "uh, no thanks" to `opam repo add`.

I'd much prefer to explicitly pin the upstream dependencies to the branches called out in the `url`s provided by the repository files; that at least limits the potential chaos to the user's current branch.